src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -873,10 +874,34 @@ protected virtual void Dispose(bool disposing)
 
         public void Dispose()
         {
+            StopWorkerProcessAsync();
             _disposing = true;
             Dispose(true);
         }
 
+        public async void StopWorkerProcessAsync()
```

Comment:
Should this need to be `public`?

Reply:
Yeah, I thought the same. I just kept it consistent with `StartWorkerProcessAsync()`. Let me know if you feel we should still update it to a private method.

https://github.com/Azure/azure-functions-host/blob/b2ed903d20416157cc8d3730246e642d3e6ed9e5/src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs#L147

Reply:
I think we should change the visibility of this method to private since it is being called only from within the same class.